### PR TITLE
Stats content card component

### PR DESCRIPTION
**Pull request by xifengjin88**
Just completed StatsContentCard component

### CODE REVIEW

**Comment by xifengjin88** [comment]
requesting for review!!

File: browser/components/Home.js [hunk 1]
```diff
@@ -27,31 +28,31 @@ class Home extends Component {
                         <table>
                             <tr>
                                 <th>Route</th>
-                                <th>Use Case</th> 
+                                <th>Use Case</th>
```
Comment (coreygreenwald) [inline browser/components/Home.js]: was this a change in white space?

File: browser/components/StatsContentCard/StatsContentCard.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './StatsContentCard.scss';
+import PropTypes from 'prop-types';
+
+
+const StatsContentCard = (props) => {
+  const { godName, imageUrl, featureStats } = props;
+
+  return (
+    <div>
```
Comment (coreygreenwald) [inline browser/components/StatsContentCard/StatsContentCard.js]: can we throw a classname on this like stats-content-card

File: browser/components/StatsContentCard/StatsContentCard.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      <h1>God name: {godName}</h1>
+      <img src={imageUrl} alt="image of god" />
+      <div>
+        {Object.keys(featureStats).map(stat => <p>{stat}: {featureStats[stat]}</p>)}
```
Comment (coreygreenwald) [inline browser/components/StatsContentCard/StatsContentCard.js]: Aren't we passing a single stat? I'm confused.

File: browser/components/StatsContentCard/StatsContentCard.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+      </div>
+    </div>
+  )
+}
+
+StatsContentCard.propTypes = {
```
Comment (coreygreenwald) [inline browser/components/StatsContentCard/StatsContentCard.js]: Nice 😎

File: browser/components/StatsContentCard/StatsContentCard.scss [hunk 5]
```diff
@@ -0,0 +1,3 @@
+.StatsContentCard {
```
Comment (coreygreenwald) [inline browser/components/StatsContentCard/StatsContentCard.scss]: This className doesn't exist and you should use kebab case! ```stats-content-card```

File: webpack.config.js [hunk 6]
```diff
@@ -2,7 +2,7 @@ module.exports = {
   entry: './browser/index.js',
   output: {
     path: __dirname,
-    filename: '../EsportsDataV2/public/bundle.js'
+    filename: '../smitedataaggregator/public/bundle.js'
```
Comment (coreygreenwald) [inline webpack.config.js]: we should set up a config for this

File: browser/components/StatsContentCard/StatsContentCard.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+import './StatsContentCard.scss';
+import PropTypes from 'prop-types';
+
+
+const StatsContentCard = (props) => {
+  const { godName, imageUrl, featureStats } = props;
+
+  return (
+    <div>
+      <h1>God name: {godName}</h1>
```
Comment (coreygreenwald) [inline browser/components/StatsContentCard/StatsContentCard.js]: ```className="stats-content-card-title"```

File: browser/components/StatsContentCard/StatsContentCard.js [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+      <img src={imageUrl} alt="image of god" />
```
Comment (coreygreenwald) [inline browser/components/StatsContentCard/StatsContentCard.js]: ```className="stats-content-card-image"```

File: browser/components/StatsContentCard/StatsContentCard.js [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+      <div>
```
Comment (coreygreenwald) [inline browser/components/StatsContentCard/StatsContentCard.js]: ```className="stats-content-card-stat"```
...until i can think of something better 🤔

File: browser/components/StatsContentCard/StatsContentCard.js (hunk 3 above)
Comment (xifengjin88) [reply browser/components/StatsContentCard/StatsContentCard.js]: that's right, will update that

File: browser/components/Home.js (hunk 1 above)
Comment (xifengjin88) [reply browser/components/Home.js]: Wasn't aware of it. Will look into it

File: browser/components/StatsContentCard/StatsContentCard.js (hunk 2 above)
Comment (xifengjin88) [reply browser/components/StatsContentCard/StatsContentCard.js]: Will do and will add more styling

**Pull request closed without merging**